src/Symfony/Bridge/ProxyManager/LazyProxy/PhpDumper/ProxyDumper.php

```diff
@@ -63,7 +63,7 @@ public function isProxyCandidate(Definition $definition)
     /**
      * {@inheritdoc}
      */
-    public function getProxyFactoryCode(Definition $definition, $id)
+    public function getProxyFactoryCode(Definition $definition, $id, $methodName = null)
```

Comment:
As child classes are allowed to add additional arguments (see https://3v4l.org/CmOt2) you could use `func_num_args()` and `func_get_arg()` to create a forward compatible layer.